target/linux/qualcommax/image/ipq807x.mk

```
@@ -195,7 +195,7 @@ define Device/prpl_haze
 	DEVICE_DTS_CONFIG := config@hk09
 	SOC := ipq8072
 	DEVICE_PACKAGES += ath11k-firmware-qcn9074 ipq-wifi-prpl_haze kmod-ath11k-pci \
-		mkf2fs f2fsck kmod-fs-f2fs
+		mkf2fs f2fsck kmod-fs-f2fs kmod-leds-lp55xx-common kmod-leds-lp5562
```

```suggestion
		mkf2fs f2fsck kmod-fs-f2fs kmod-leds-lp5562
```

`kmod-leds-lp5562` already depends on `kmod-leds-lp55xx-common`